extensions/2.0/Vendor/EXT_feature_metadata/schema/class.property.schema.json

```diff
@@ -62,39 +62,33 @@
       "default": false
     },
     "max": {
-      "type": [
-        "number",
-        "array"
+      "oneOf": [
+        {"type": "number"},
+        {"type": "array", "items": {"type": "number"}, "minItems": 1}
       ],
-      "items": {
-        "type": "number"
-      },
       "description": "Maximum allowed values for property values. Only applicable for numeric types and fixed-length arrays of numeric types. For numeric types this is a single number. For fixed-length arrays this is an array with `componentCount` number of elements. The `normalized` property has no effect on these values: they always correspond to the integer values."
```

Comment:
We mention fixed-length arrays here, but this would also apply for vectors and matrices.

Reply:
Done ✅